test/packages/benchmarks/rally_benchmark/_dev/benchmark/rally/logs-benchmark.yml

```diff
@@ -0,0 +1,20 @@
+---
+description: Benchmark 20000 events ingested
+input: filestream
+vars: ~
+data_stream:
+  name: testds
+  vars:
+    paths:
+      - "dummy path"
+warmup_time_period: 10s
+corpora:
+  generator:
+    total_events: 20000
+    template:
```

Comment:
I'm looking at the templates from kubernetes.container: https://github.com/elastic/elastic-integration-corpus-generator-tool/tree/main/assets/templates/kubernetes.container There are two `.tpl` files. How does this translate to this structure? 

There is also configs and config, is that a cleanup?

Reply:
> I'm looking at the templates from kubernetes.container: https://github.com/elastic/elastic-integration-corpus-generator-tool/tree/main/assets/templates/kubernetes.container There are two `.tpl` files. How does this translate to this structure?

the two template are the same. one "pretty-printed" with new lines. we will use only the latter once migrating it to the package (currently `elastic-package` removes the newline, but we'll later do that directly in the corpus generator)
 
> There is also configs and config, is that a cleanup?

I see only `configs.yaml`. there will be only one config file per benchmark. how do we name it it's not relevant, as long as we refer it with the proper name in the benchmark scenario, and while having some naming convention might be useful :)

Reply:
++ on having a convention. If we have the convention and someone follows it, it means also the config could be left out by default eventually. Something we can iterate on.